assets/js/docs-feedback.js

```diff
@@ -0,0 +1,75 @@
+/**
+ *
+ */
+$(document).ready(function(){
+    // Check that the analytics track function is available.
+    var analyticsAvailable = window.analytics && window.analytics.track &&
+                             (typeof window.analytics.track === "function");
+
+    if (analyticsAvailable) {
+        // Show the docs feedback container.
+        $("#docsFeedbackContainer").removeClass("hidden");
+
+        // For each of the button ids attach a click hanlder that shows
+        // the additional comment section.
+        ["#docsFeedbackYes", "#docsFeedbackNo"].forEach(function(key) {
+            var answer = key === "#docsFeedbackYes" ? "Yes" : "No";
+            $(key).on("click", function() {
+                $("#feedbackButtons").addClass("hidden");
+                $("#feedbackLongForm").removeClass("hidden");
+
+                // Show the additional comment section.
+                showAdditionalCommentSection(answer);
+            });
+        });
+
+        /**
+         * Submit the documentation feedback to Segment.
+         *
+         * @param {string} answer The value of the button the user clicked when giving feedback.
+         * @param {string} comments The value of the text area in the comment section. Defaults to an empty string
+         * @param {string} email The value of the email input in the comment section. Defaults to an empty string
+         */
+        function sendFeedbackToSegement(answer, comments = "", email = "") {
```

Comment:
Default parameters are an ES2015 feature, which won't work in IE. I don't think we really care much about IE at this point, but it'd still be nice if the site worked (for the most part) there.

As an aside, if we continue to invest in scripting on the site, I'd love to setup a more modern pipeline, so we can use modern language features, TypeScript, webpack, etc.